### PR TITLE
Feature: onboarding PEVA organisations

### PR DESCRIPTION
**Pull request by mirdono**
OP-2640, OP-2859, OP2860, OP-2863, OP-2862, OP-2864

Prepare the frontend to handle, the to be onboarded, organisations of type PEVA municipality and PEVA province.

Backend PR: https://github.com/lblod/app-organization-portal/pull/368
This should be tested against v0.1.7 of the [construct-administrative-unit-relationships-service](https://github.com/lblod/construct-administrative-unit-relationships-service).

### CODE REVIEW

**Review by aliokan — COMMENTED** [review]
@mirdono Except for the `isPeva` duplication but I can live with until all forms migrate to the new validation. The rest looks good :+1:

File: app/controllers/administrative-units/administrative-unit/core-data/edit.js [hunk 1]
```diff
@@ -24,6 +24,20 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataEditController
...
+    );
+  }
+
+  get isPevaProvince() {
+    return (
+      this.model.administrativeUnit.classification?.get('id') ===
+      CLASSIFICATION_CODE.PEVA_PROVINCE
+    );
+  }
+
```
[the remark this answers is not in the document]
Comment (mirdono) [reply app/controllers/administrative-units/administrative-unit/core-data/edit.js]: Good catch, I appear to have missed these when merging the development branch.

**Comment by mirdono** [comment]
> @mirdono Except for the `isPeva` duplication but I can live with until all forms migrate to the new validation. The rest looks good 👍

@aliokan I removed most of the `isPeva*` functions. The only ones remaining (except those in the model) are in the [site-type-select ](https://github.com/lblod/frontend-organization-portal/blob/4ac41da79993fce7979f201e1c619427456b486f/app/components/site-type-select.js) component as I think this requires changing the component's implementation, and in the [soon to be deprecated validations](https://github.com/lblod/frontend-organization-portal/blob/4ac41da79993fce7979f201e1c619427456b486f/app/validations/administrative-unit.js)

**Review by aliokan — APPROVED** [review]
Look good! :pray:

**Pull request merged**